app/controllers/users_controller.rb

```diff
@@ -1,5 +1,12 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+    @posts = Post.where(author_id: params[:id])
+    @comments = Comment.where(posts_id: params[:id])
+  end
```

Comment:
* Based on the wireframe provided your user-controller index page is supposed to render 3 most recent posts as shown in the wireframe below 👇👇 **HINT** 💭  you can create a method that will fetch the 3 most recent posts
![image](https://user-images.githubusercontent.com/52497246/214910887-a566b4f7-5ced-41c0-8ce4-94b3e6487136.png)